packages/react-native/sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -23,7 +23,7 @@ end
 package = JSON.parse(File.read(File.join(react_native_path, "package.json")))
 # [macOS
 rn_version = package['version']
-version = findLastestVersionWithArtifact(rn_version) || rn_version
+version = findLatestVersionWithArtifact(rn_version) || rn_version
```

Comment:
I think this is out of scope / a second PR, but I would like to revisit our "just get the latest version of Hermes for a given release (0.78.x) and be a little more exact. We've talked about this offline though